app/models/async_transaction/evss/va526ez_submit_transaction.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module AsyncTransaction
+  module EVSS
+    class VA526ezSubmitTransaction < AsyncTransaction::Base
+      JOB_STATUS = {
+        submitted: 'submitted',
+        received: 'received',
+        retrying: 'retrying',
+        non_retryable_error: 'non_retryable_error',
+        exhausted: 'exhausted'
+      }.freeze
+      SOURCE = 'EVSS'
+
+      scope :for_user, ->(user) { where(user_uuid: user.uuid) }
+      scope :job_id, ->(job_id) { where(transaction_id: job_id).limit(1) }
```

Comment:
It should be limited to one result and job_ids are unique, I think I want `.first` rather than `limit(1)` so it returns a single record rather than an array